### DISCUSSION TITLE
swagger-ts: custom transform enum keys convention

### DISCUSSION THREAD

**Discussion by raidenmiro** [kubb-labs/kubb#1077] [Q&A]
I am using the `@kubb/swagger-ts` plugin to generate types. Is there an easy way to convert enum keys to PascalCase?

For example, converting:

```ts
export enum Foo {
    "bar" = "bar",
    "foo_bar" = "foo_bar"
}
```

into:

```ts
export enum Foo {
  Bar = 'bar',
  FooBar = 'foo_bar'
}
```

So far, I see a solution involving the `typescript-ast` (https://nabeelvalley.co.za/docs/javascript/typescript-ast/), in the form of a separate script that will execute on `hooks.done`. Maybe I'm missing something and there are better solutions?

**Comment by raidenmiro** [comment]
How good an idea is it to give transform keys via transformers.pascalCase() https://github.com/kubb-labs/kubb/blob/ffdf3509444370b8a49e4af21b4877bca6d4e3d6/packages/plugin-oas/src/SchemaGenerator.ts#L634? 🤔

**Reply by Samuel-Morgan-Tyghe** [reply]
https://github.com/prettier/eslint-plugin-prettier/issues/306
solves some eslint issues aswell
